robyn/argument_parser.py

```diff
@@ -87,6 +88,8 @@ def __init__(self) -> None:
             if arg.endswith(".py"):
                 self.file_path = arg
                 break
+        else:
+            self.file_path = sys.argv[0]
```

Comment:
this is clever. but where are you using this?

Reply:
in `EventHandler#reload` (inside the new `if` block I added)